eng/common/scripts/Helpers/DevOps-WorkItem-Helpers.ps1

```diff
@@ -490,21 +545,45 @@ function CreateOrUpdatePackageWorkItem($lang, $pkg, $verMajorMinor, $existingIte
       }
     }
 
-    $newparentItem = FindOrCreatePackageGroupParent $serviceName $pkgDisplayName -outputCommand $false
+    $newparentItem = FindOrCreateReleasePlanParent $serviceName $pkgDisplayName -outputCommand $false
     UpdateWorkItemParent $existingItem $newParentItem -outputCommand $outputCommand
     return $existingItem
   }
 
-  $parentItem = FindOrCreatePackageGroupParent $serviceName $pkgDisplayName -outputCommand $false
+  $parentItem = FindOrCreateReleasePlanParent $serviceName $pkgDisplayName -outputCommand $false
   $workItem = CreateWorkItem $title "Package" "Release" "Release" $fields $assignedTo $parentItem.id -outputCommand $outputCommand
   Write-Host "[$($workItem.id)]$lang - $pkgName($verMajorMinor) - Created"
   return $workItem
 }
 
+function FindOrCreateReleasePlanParent($serviceName, $packageDisplayName, $outputCommand = $true)
+{
+  $existingItem = FindReleasePlanWorkItem $serviceName $packageDisplayName -outputCommand $outputCommand
+  if ($existingItem) {
+    Write-Host "Found existing release plan work item [$($existingItem.id)]"
```

Comment:
Coul a test on Dev triggers this function?
Wondering what would happen if this gets trigger and then it doesn't find the existing work item because it has the test tag.

Reply:
Can you explain what do you mean by running on Dev?

Reply:
If an existing release plan is not found, then it will create a new one. But this gets triggered only when someone runs prepare release script and it should always exclude test work items.